handbook/product/roles/product_designer.md

```diff
@@ -57,10 +59,10 @@ Learn more about what it is like to work at Sourcegraph by reading [our handbook
    - [30 min] [HM intro call](./interviews/hm_intro_call.md) with [Christina Forney](../../../company/team/index.md#christina-forney-she-her) (VP Product)
 1. [1 hr] **[Portfolio review](./interviews/pd_portfolio_review.md)**: you to present your past work.
```

Comment:
```suggestion
1. [1 hr] **[Portfolio review](./interviews/pd_portfolio_review.md)**: You share a case study demonstrating your past work.
```

Reply:
Quinn, I think this is a great addition. A small improvement is here: https://github.com/sourcegraph/about/pull/1599#discussion_r491182816